src/CommandingOfficers/TrilogyCommander.java

```diff
@@ -0,0 +1,85 @@
+package CommandingOfficers;
+
+import Engine.GameScenario;
+import Engine.UnitMods.UnitDamageModifier;
+import Engine.UnitMods.UnitModifier;
+import Terrain.MapMaster;
+import Units.Unit;
+import Units.UnitDelta;
+import java.util.ArrayList;
+import CommandingOfficers.CommanderInfo.InfoPage;
+
+/** Base class for AW2/3 COs (AW1 is special) */
+public abstract class TrilogyCommander extends Commander
+{
+  private static final long serialVersionUID = 1L;
+  public static final InfoPage TRILOGY_MECHANICS_BLURB = new InfoPage(
+      "CO powers never charge while a power is active.\n"
+    + "On each power activation, each star in your meter costs 1/5 more of the base star cost.\n"
+    + "This continues for 9 activations, after which the star cost settles at double the base star cost.\n"
+    + "CO power activation deducts the *new* power cost rather than the old one.\n"
```

Comment:
Wait, what? That seems wrong. Do the carts actually do that? Might explain some things.

Reply:
This is also a known AWBW difference.
And yes, it's very trippy that Hachi can COP twice with no extra charge for his 9th and 10th casts (on his 3/5 meter)

Reply:
But only on the 9th and 10th casts? That seems odd either way, since this nuance should always work against the player.

Reply:
Yes. Cast 9 costs 2.8x base, cast 10 costs 2.0x base. It's a bug that the cost ever goes down.
So you get to keep more than (for cast 10) two stars' worth of charge, then you get a refund on the energy you _did_ spend.
Cast 11 costs exactly the same as cast 10 (so Hachi keeps exactly 2* if he charged SCOP before COPing)

...and I know how this happened because I accidentally did it myself.

Reply:
...and now we are doing it on purpose for verisimilitude. Cool I guess.